lib/prompts/expand.js

```diff
@@ -115,7 +115,10 @@ Prompt.prototype.getCurrentValue = function (input) {
     return null;
   }
 
-  return selected.value || selected.name;
+  if (selected.value === undefined) {
```

Comment:
Why change this line? The new code does the same thing in 4 lines instead of one.